cocos/render-scene/core/pass.ts

```diff
@@ -845,8 +834,16 @@ export class Pass {
     get blocks (): Float32Array[] { return this._blocks; }
     get blocksInt (): Int32Array[] { return this._blocksInt; }
     get rootBufferDirty (): boolean { return this._rootBufferDirty; }
+    /**
+     * @engineInternal
+     */
+    set rootBufferDirty (val: boolean) { this._rootBufferDirty = val; }
     // states
     get priority (): RenderPriority { return this._priority; }
+    /**
+     * @engineInternal
+     */
+    set priority (val: RenderPriority) { this._priority = val; }
```

Comment:
unfortunately, the `engineInternal` cannot only apply to setter or getter, 
if we apply `engineInternal` to a setter, then getter is `engineInternal` too

can we provide another method like 
```ts
/**
 * @engineInternal
 */
setRootBufferDirty (val: boolean) { this._rootBufferDirty = val; }
/**
 * @engineInternal
 */
setPriority (val: RenderPriority) { this._priority = val; }
```

Reply:
Ok. It is changed to function now.